docs/content/intro/quickstart.md

```diff
@@ -2,43 +2,265 @@
+**linux**
+```bash
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-linux-amd64
+chmod +x /usr/local/bin/brig
+```
 
+**macos**
 ```bash
-# add Brigade chart repo
-helm repo add brigade https://brigadecore.github.io/charts
-# install Brigade
-helm install -n brigade brigade/brigade
+curl -Lo /usr/local/bin/brig https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-darwin-amd64
+chmod +x /usr/local/bin/brig
+```
 
-# if you want to activate Generic Gateway, you should use this command
-# helm install -n brigade brigade/brigade --set genericGateway.enabled=true
+**windows**
+```powershell
+mkdir -force $env:USERPROFILE\bin
+(New-Object Net.WebClient).DownloadFile("https://github.com/brigadecore/brigade/releases/download/v2.0.0-alpha.5/brig-windows-amd64.exe", "$ENV:USERPROFILE\bin\brig.exe")
+$env:PATH+=";$env:USERPROFILE\bin"
 ```
 
-You will now have Brigade installed. [Kashti](https://github.com/brigadecore/kashti), the dashboard for your Brigade pipelines, is also installed in the cluster.
+The script above downloads brig.exe and adds it to your PATH for the current session.
+Add the following line to your [PowerShell Profile](https://www.howtogeek.com/126469/how-to-create-a-powershell-profile/) to make the change permanent.
 
-## Install brig
+```powershell
+$env:PATH+=";$env:USERPROFILE\bin"
+```
 
-Brig is the Brigade command line client. You can use `brig` to create/update/delete new brigade Projects, run Builds, etc. To get `brig`, navigate to the [Releases page](https://github.com/brigadecore/brigade/releases/) and then download the appropriate client for your platform. For example, if you're using Linux or WSL, you can get the 1.2.1 version in this way:
+## Install Brigade
 
-```bash
-wget -O brig https://github.com/brigadecore/brigade/releases/download/v1.2.1/brig-linux-amd64
-chmod +x brig
-mv brig ~/bin
+Install Brigade on your local development cluster. See our [Installation] instructions for full instructions suitable for production clusters.
+
+1. Enable Helm's experimental OCI support by setting the `HELM_EXPERIMENTAL_OCI` environment variable to 1.
+
+    **bash**
+    ```bash
+    export HELM_EXPERIMENTAL_OCI=1
+    ```
+
+    **powershell**
+    ```powershell
+    $env:HELM_EXPERIMENTAL_OCI=1
+    ```
+
+1. Create a directory to store the Brigade Helm charts.
+
+    **bash**
+    ```bash
+    mkdir -p ~/charts
+    ```
+
+    **powershell**
+    ```powershell
+    mkdir -force $env:USERPROFILE/charts
+    ```
+
+1. Run the following commands to install Brigade.
+
+    ```
+    helm chart pull ghcr.io/brigadecore/brigade:v2.0.0-alpha.5
+    helm chart export ghcr.io/brigadecore/brigade:v2.0.0-alpha.5 -d ~/charts
+    kubectl create namespace brigade2
+    helm install brigade2 ~/charts/brigade --namespace brigade2
+    kubectl rollout status deployment brigade2-apiserver -n brigade2 --timeout 5m
```

Comment:
I think these three commands can reasonably be condensed into:

```
helm install brigade2 ~/charts/brigade --namespace brigade2 --create-namespace --wait`
```

Reply:
Sadly --wait doesn't work. That command will wait until the timeout. I think helm is looking at more than just the deployment rollout status (maybe a service ip address being assigned?) and never considers it "done".

I can smoosh in --create-namespace though!

Reply:
Ya... you're right... and I remember now...

It's because we default to a service of type `LoadBalancer`.

@vdice and I previously went back and forth on whether that was a sensible default or not.

I guess we should, again, reconsider that.